lib/galaxy_test/selenium/test_history_dataset_state.py

```diff
@@ -63,10 +63,12 @@ def _prepare_dataset(self, hid):
         # Expand HDA and wait for details to show up.
         return self.history_panel_click_item_title(hid=hid, wait=True)
 
-    def _assert_title_buttons(self, hid, expected_buttons=['display', 'edit', 'delete']):
+    def _assert_title_buttons(self, hid, expected_buttons=None):
+        expected_buttons = expected_buttons or ['display', 'edit', 'delete']
```

Comment:
```suggestion
        if expected_buttons is None:
            expected_buttons = ['display', 'edit', 'delete']
```

or it won't be possible to pass `[]`.